package.json

```diff
@@ -21,7 +21,7 @@
     "test": "run-s test:*",
     "test-ci": "jest --ci --coverage",
     "test:js": "jest",
-    "test:vrt": "build-storybook && loki --require-reference --reactUri file:./storybook-static --storiesFilter=virtru/.*",
+    "test:vrt": "build-storybook && loki --require-reference --reactUri file:./storybook-static --storiesFilter=virtru/.* --chromeDockerImage=yukinying/chrome-headless-browser-stable",
```

Comment:
Can you create a story and add a `TODO` here to upgrade to `loki` dependency when we are able to? It looks like this will be coming soon in the next update: https://github.com/oblador/loki/commit/969989780a72241a84852c35938c21377c8dfae6

Reply:
I had to rollback this change because it doesn't work with our CI